assets/dev/js/editor/elements/views/behaviors/sortable.js

```diff
@@ -102,16 +102,23 @@ SortableBehavior = Marionette.Behavior.extend( {
 	startSort: function( event, ui ) {
 		event.stopPropagation();
 
-		var model = this.view.collection.get( {
-			cid: ui.item.data( 'model-cid' ),
-		} );
+		const container = elementor.getContainer( ui.item.attr( 'data-id' ) ) || {};
+
+		// BC?, handles issue with navigator sorting.
+		if ( ! Object.keys( container ).length ) {
```

Comment:
Remove BC. `getContainer` should always return the exist container or create one.

Reply:
Handled